src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2246,7 +2246,8 @@ because xbuild doesn't support framework reference assemblies.
     AndroidEmbedProfilers="$(AndroidEmbedProfilers)"
     AndroidGdbDebugServer="$(AndroidGdbDebugServer)"
     HttpClientHandlerType="$(AndroidHttpClientHandlerType)"
-    TlsProvider="$(AndroidTlsProvider)">
+    TlsProvider="$(AndroidTlsProvider)"
+    EnableSGenConc="$(EnableSGenConc)">
```

Comment:
We normally named properties with an `Android` prefix, and avoid abbreviations, thus `$(AndroidEnableSgenConccurrentGC)`.

That said, I'm not certain I see why we need this change:

* It changes the default from what `mono` thinks the default should be. Is that wise?
* It introduces a new MSBuild property. Typically this is only necessary if the IDE is going to set it -- MSBuild properties are the primary "communication" mechanism between the IDEs and our build system. If the IDE isn't getting e.g. a checkbox for this, then the need for a property is reduced.
* This can already be specified by the end user, via the `MONO_GC_PARAMS` environment variable.

What is the intended use case for this property?